xclim/testing/tests/test_ensembles.py

```diff
@@ -68,17 +69,25 @@ def test_create_ensemble(self):
                 ens1.isel(realization=i).tg_mean.values, ds_all[i].tg_mean.values
             )
 
-    def test_no_time(self):
+    def test_no_time(self, tmpdir):
```

Comment:
I think `tmp_path` has replaced `tmp_dir` as thre suggested way to go, but not sure.

Reply:
done ..